docs/03-dev-howtos/01-ab-testing.md

```diff
@@ -6,26 +6,35 @@ Most tests can be written in JavaScript, although we can serve variants via Varn
 
 Read this if you want to [write a server-side AB Test](#write-a-server-side-test)
 
-# Guide
+## Guide
 
 There are six steps in the test lifecycle:
 
- - [Adding a switch to turn the test on & off](#adding-a-switch)
- - [Writing a test, which is typically a simple JS module](#writing-a-test)
- - [Running the test](#running-the-test)
+ - [Adding a switch to turn the test on & off)](#adding-a-switch). For server-side tests, this is included in the test definition.
```

Comment:
```suggestion
 - [Adding a switch to turn the test on & off)](#adding-a-switch). For server-side tests, the fallback state is included in the test definition.
```

I think

Reply:
My understanding is when you create a server-side test it automatically creates an associated switch, and because it extends from the `Switch` class the `Experiment` requires to have initial values for the Switch defined.

Reply:
Maybe “automatically created” by the test definition? But yes basically I think your wording is more correct than @alinaboghiu 😀